### DISCUSSION TITLE
Web UI doesn't show all the information about connected users

### DISCUSSION THREAD

**Discussion by dennispalwanow** [wg-easy/wg-easy#419] [Q&A]
Hello! Sorry for bothering you. Please could someone tell me how can I get information about all the connected users in Web UI, current I see the info only first user config(screenshot attached)? Because other users also are using configs, but I don't see any information about Internet traffic.
<img width="679" alt="screenshot" src="https://user-images.githubusercontent.com/84556661/202202240-a32c48f8-3463-4865-a4a8-0c9a83af4c6b.png">

**Comment by mitrios-hub** [comment]
У вас интерфейс нормально работает? Юзеры онлайн отображаются красными точками и показывается трафик
![изображение](https://user-images.githubusercontent.com/52719813/209345255-e7b05dfb-851d-4faa-9231-e2203568e656.png)
